swift/test/Ice/objects/TestI.swift

```diff
@@ -250,15 +250,16 @@ class F2I: F2 {
     func op(current _: Current) async throws {}
 }
 
-class UnexpectedObjectExceptionTestI: Ice.Blobject {
-    func ice_invoke(inEncaps _: Data, current: Ice.Current) throws -> (ok: Bool, outParams: Data) {
+class UnexpectedObjectExceptionTestI: Ice.Dispatcher {
```

Comment:
Since UnexpectedObjectExceptionTestI is not a dispatcher, it would be clear to call it
`UnexpectedObjectExceptionTestDisp`.